arcgis-ios-sdk-samples/Layers/Query with CQL filters/QueryWithCQLFiltersViewController.swift

```diff
@@ -0,0 +1,122 @@
+    let ogcFeatureLayer: AGSFeatureLayer = {
+        // Note: the collection ID can be accessed later via
+        // `featureCollectionInfo.collectionID` property of the feature table.
+        let table = AGSOGCFeatureCollectionTable(
+            url: URL(string: "https://demo.ldproxy.net/daraa")!,
+            collectionID: "TransportationGroundCrv"
+        )
+        // Set the feature request mode to manual (only manual is currently
+        // supported). In this mode, you must manually populate the table -
+        // panning and zooming won't request features automatically.
+        table.featureRequestMode = .manualCache
+        
+        let featureLayer = AGSFeatureLayer(featureTable: table)
+        let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .red, width: 3)
+        featureLayer.renderer = AGSSimpleRenderer(symbol: lineSymbol)
+        return featureLayer
+    }()
+    
+    // MARK: Method
+    
+    func populateFeaturesFromQuery(queryParameters: AGSQueryParameters) {
+        // Cancel if there is an existing query request.
+        lastQuery?.cancel()
+        
+        // Populate the table with the query. Setting `outFields` to `nil`
+        // requests all fields.
+        let table = ogcFeatureLayer.featureTable as! AGSOGCFeatureCollectionTable
+        lastQuery = table.populateFromService(
+            with: queryParameters,
+            clearCache: true,
+            outfields: nil
+        ) { [weak self] result, error in
+            guard let self = self else { return }
+            if let error = error,
+               // Do not display error if user cancelled the request.
+               (error as NSError).code != NSUserCancelledError {
+                self.presentAlert(error: error)
+            } else if let result = result, let extent = self.ogcFeatureLayer.featureTable?.extent {
+                // Zoom to the extent of the selected collection.
+                self.mapView.setViewpointGeometry(extent, padding: 50, completion: nil)
+                self.presentAlert(title: "Query Result", message: "Query returned \(result.featureEnumerator().allObjects.count) features.")
+            }
+        }
+    }
+    
+    // MARK: UIViewController
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Add the source code button item to the right of navigation bar.
+        (navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = [
+            "QueryWithCQLFiltersViewController",
+            "QueryWithCQLFiltersSettingsViewController"
+        ]
+        ogcFeatureLayer.load { [weak self] error in
+            guard let self = self else { return }
+            if let error = error {
+                self.presentAlert(error: error)
+            } else {
+                // Show a default query result.
+                let defaultQueryParameters = AGSQueryParameters()
+                defaultQueryParameters.maxFeatures = 1_000
```

Comment:
Since the initial query is set to 1000 max features, I think the table view should display the default number upon loading as well. A user might not know to tap on "Max features" and having the default value displayed there may help.